skills/common/process/batch/runtime/powershell/Scripts/Invoke-BatchOrchestrator.ps1

```
@@ -94,6 +95,79 @@ function Set-BatchStateTransition {
+}
+
+function Test-GitBranchExists {
+    param(
+        [string]$BranchName,
+        [string]$Repository = "."
+    )
+    & git -C $Repository rev-parse --verify --quiet "refs/heads/$BranchName" > $null 2>&1
+    return $LASTEXITCODE -eq 0
+}
+
+function Test-GitPrExists {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Test-GitPrExists' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Test-GitPrExists'

(PSUseSingularNouns)